man/README.md

```diff
@@ -23,11 +24,11 @@ Using this `Dockerfile`, create a Docker image tagged `docker/md2man`:
 
 Once the image is built, run a container using the image with *volumes*:
 
-    docker run -v /<path-to-git-dir>/docker/man:/docs:rw \
+    docker run -v /<path-to-git-dir>/<docker-fork>/man:/docs:rw \
```

Comment:
Given that the "building the md2man image" works from within the `/man` directory, perhaps we can use something like;
```Markdown

    docker run -v $(pwd):/man -w /man -i docker/md2man md2man-all.sh
```
I simplified/changed the instruction a bit;
- using `/man` as the container path (for consistency)
- dropped `:rw` for the volume (it's the default)
- dropped the path to `md2man-all.sh` (because there's already a `-w` for the workdir)

Wdyt? Should we add an extra "from within the `man` directory, run this command:" (or similar)?

(Note; I just wrote the command down, you might want to test it just to be sure.)